transformer.py

```diff
@@ -183,18 +197,18 @@ def add_parameters(parser: argparse.ArgumentParser) -> None:
                         default="Unknown",
                         help="name of the germplasm associated with the canopy cover")
 
-#pylint: disable=unused-argument
-def check_continue(transformer: transformer_class.Transformer, check_md: dict, transformer_md: dict, full_md: dict) -> tuple:
+
+# pylint: disable=unused-argument
+def check_continue(check_md: dict) -> tuple:
     """Checks if conditions are right for continuing processing
     Arguments:
-        transformer: instance of transformer class
     Return:
```

Comment:
Add documentation on all parameters (once they're restored) can be copied from another transformer